hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -194,24 +195,23 @@ vcgencmd otp_dump | grep 28: | sed s/.*://g
 ```
 
 ### Installation - firmware update
-Network boot functionality requires the 2020-02-13 Raspbian Buster release (or newer) and an updated bootloader. For advanced boot modes (USB, network) it is normally best to use the latest 'stable' bootloader.
+Network boot is available in the latest production bootloader (pieeprom-2020-04-16.bin). Previous beta/stable releases which supported network boot are still available but are deprecated.
 
 ```
 # Update the rpi-eeprom package                                                                                                 
 sudo apt update
 sudo apt upgrade
-                                                                          
-# Network boot is only available in the stable and beta releases.
-# Switch from critical updates to the stable release series.
-# As root:-
-echo FIRMWARE_RELEASE_STATUS="stable" > /etc/default/rpi-eeprom-update
+# Check the current version
+sudo rpi-eeprom-update     
+# Update to latest
+sudo rpi-eeprom-update     
```

Comment:
Is this correct? It seems to be showing the same command for both "Check the current version" and "Update to latest" ??